docs/mkdocs.yml

```diff
@@ -66,7 +67,6 @@ markdown_extensions:
   - pymdownx.magiclink
   - pymdownx.mark
   - pymdownx.details
-  - pymdownx.superfences
```

Comment:
@pgrunewald any particular reason you removed pymdownx.superfences?

Reply:
@tisto: Because it was duplicated in the extension list. Unfortunately the diff view in GitHub didn't quite show this.

Reply:
@pgrunewald ah, cool! Did not see this. Thanks for cleaning up! :)